src/rgw/rgw_zone.cc

```diff
@@ -2059,10 +2068,12 @@ int RGWZoneGroupTierS3Glacier::update_params(const JSONFormattable& config)
   if (config.exists("glacier_restore_tier_type")) {
     string s;
     s = config["glacier_restore_tier_type"];
-    if (s != "Expedited") {
+    if (s == "Expedited") {
+      glacier_restore_tier_type = Expedited;
+    } else if (s == "Standard") {
       glacier_restore_tier_type = Standard;
     } else {
-      glacier_restore_tier_type = Expedited;
+      glacier_restore_tier_type = NoTier;
```

Comment:
nit: I think it may be good to have "Standard" in `else` case as that's the default value for `glacier_restore_tier_type`.